app/controllers/repositories_controller.rb

```diff
@@ -1,7 +1,10 @@
 class RepositoriesController < ApplicationController
-  after_action :mark_everything_as_read, only: [:index, :show]
+  after_action :mark_everything_as_read, only: [:index]
 
-  @client = Octokit::Client.new(:access_token => ENV['PERSONAL_ACCESS_TOKEN'])
+  def initialize
```

Comment:
See other point about using a `def client` :), - typically `initialize` is never overridden for controllers or models - I'd say `before_action`s and lazy ivars are more common for setup as we don't really control when a controller is initialized.